### PR TITLE
[ADD] pos_disable_change_cashier

### PR DESCRIPTION
**Pull request by legalsylvain**
Proposing new module ``pos_disable_change_cashier``

## Point of Sale - Disable Change Cashier

This module extends Odoo Point Of Sale features, allowing to disable the button that allow to switch cashiers.

It can be interested in small shops and / or in multi company context, when the pop up displays a lot of users that are not cashiers.

### Configuration

Once installed, the button is disabled by default.

You can enable it in the point configuration.

pos_config_form

@sandiefavre, @quentinDupont 

#GRAPOCA

### CODE REVIEW

File: pos_disable_change_cashier/views/pos_config.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+<!--
+    Copyright (C) 2020 - Today: GRAP (http://www.grap.coop)
+    @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<odoo>
+
+    <record model="ir.ui.view" id="pos_config_view_form">
+        <field name="name">pos.config.form (in pos_default_partner)</field>
```
Comment (ivantodorovich) [inline pos_disable_change_cashier/views/pos_config.xml]:
```suggestion
```

**Review by ivantodorovich — APPROVED** [review]

File: pos_disable_change_cashier/views/pos_config.xml (hunk 1 above)
Comment (legalsylvain) [reply pos_disable_change_cashier/views/pos_config.xml]: thanks ! (bad copy / paste) ;-)

File: pos_disable_change_cashier/i18n/fr.po [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: pos_disable_change_cashier
+#: model:ir.model.fields,field_description:pos_disable_change_cashier.field_pos_config__iface_change_cashier
+msgid "Change Cashier"
+msgstr "Changer de caisser"
```
Comment (SandieFavre) [inline pos_disable_change_cashier/i18n/fr.po]: typo : "Changer de caissier"

**Review by SandieFavre — APPROVED** [review]
LGTM (functional review)

File: pos_disable_change_cashier/i18n/fr.po (hunk 2 above)
Comment (legalsylvain) [reply pos_disable_change_cashier/i18n/fr.po]: thanks, done.

**Pull request merged**